vulkan_generator/tests/test_types.py

```diff
@@ -22,8 +22,8 @@
 
 import xml.etree.ElementTree as ET
 
-from vulkan_parser import handle_parser
-from vulkan_parser.types import VulkanHandle, VulkanHandleAlias
+from vulkan_parser import handle_parser, struct_parser
+from vulkan_parser.types import VulkanHandle, VulkanHandleAlias, VulkanStruct, VulkanStructAlias
```

Comment:
Should we be importing all of vulkan_parser.types here? Or if we have to import multiple things in isolation should we import them one per line? ie 

from vulkan_parser.types import VulkanHandle
from vulkan_parser.types import VulkanHandleAlias
...

What we have above is not scalable if we end up importing 12 different classes

Reply:
:+1: